### PR TITLE
Add docs to introduce Chaos Mesh Release Cycle

### PR DESCRIPTION
**Pull request by cwen0**

### CODE REVIEW

File: docs/release-cycle.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+- keep maintaining the tracking documentation for next coming release
+- communicate with contributors asking for help to complete/prune the unfinished features/docs; Or prune them by yourself if we lost the communication with contributors.
+- cut the release branch
+- draft the Release Notes for the minor version
+- release the alpha, beta, minor versions
+- keeping enriching release-related documents
+
+### Becoming Release Manager?
+
+Chaos Mesh committers and maintainers would nominate new release managers at about the 1st week of the new release cycle, the nomination should be published in slack channel #chaos-mesh-maintainers. And they would be selected if there are no more than half the number of oppositions. At last, we would announce the release managers on the document website and slack channel [#project-chaos-mesh](https://cloud-native.slack.com/archives/C0193VAV272).
```
Comment (Oreoxmt) [inline docs/release-cycle.md:40]: #chaos-mesh-maintainers 需要加一个链接吗

Comment (cwen0) [reply docs/release-cycle.md:40]: 这个是私有的，加了链接也没啥用😂

**Comment by STRRL** [comment]
/cc @YangKeao @Hexilee 

cc to other release managers

PTAL

File: docs/release-2.3-tracking.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- [Chaos Mesh release cycle](release-cycle.md)
+
+## Tracking Docs
+
+- [Release 2.3 Milestone](https://github.com/chaos-mesh/chaos-mesh/milestone/10)
+- [Docs Milestone](https://github.com/chaos-mesh/website/milestone/2)
+
+## Release manager
+
+- [STRRL](https://github.com/STRRL)
```
Comment (STRRL) [inline docs/release-2.3-tracking.md:16]: I think we'd better also append @YangKeao @Hexilee @cwen0

File: docs/release-2.3-tracking.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+| When | Week | As Developer | As Release Manager | Notes |
+| :-: | :-: | :-: | :-: | :-: |
+| Monday 6 June 2022 | Week 1 | Normal Dev | Normal Dev |  |
+| Monday 13 June 2022 | Week 2 | Normal Dev | Normal Dev |  |
+| Monday 20 June 2022 | Week 3 | Normal Dev | Normal Dev |  |
+| Monday 27 June 2022 | Week 4 | Normal Dev | Normal Dev |  |
+| Monday 4 July 2022 | Week 5 | Normal Dev | Normal Dev |  |
+| Monday 11 July 2022 | Week 6 | Code Freeze; Fix bugs | Code Freeze |  |
+| Monday 18 July 2022 | Week 7 | Code Freeze; Fix bugs | Cut new branch release 2.3 | Cleanup the CHANGELOG on master; Cherry pick bugfixes |
+| Monday 25 July 2022 | Week 8 | Fix bugs | Release Minor Version |  |
```
Comment (STRRL) [inline docs/release-2.3-tracking.md:29]:
```suggestion
| When | Week | As Developer | As Release Manager | Notes |
| :-: | :-: | :-: | :-: | :-: |
| Monday 6 June 2022 | Week 1 | Normal Dev | Normal Dev |  |
| Monday 11 July 2022 | Week 6 | Code Freeze; Fix bugs | Code Freeze |  |
| Monday 18 July 2022 | Week 7 | Code Freeze; Fix bugs | Cut new branch release 2.3 | Cleanup the CHANGELOG on master; Cherry pick bugfixes |
| Monday 25 July 2022 | Week 8 | Fix bugs | Release Minor Version |  |
```

Comment (STRRL) [reply docs/release-2.3-tracking.md:29]: I prefer to remove the weeks 2 - 5 because they are the same :)

File: docs/release-2.3-tracking.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+## Tracking Docs
+
+- [Release 2.3 Milestone](https://github.com/chaos-mesh/chaos-mesh/milestone/10)
+- [Docs Milestone](https://github.com/chaos-mesh/website/milestone/2)
+
+## Release manager
+
+- [STRRL](https://github.com/STRRL)
+
+## Timeline
```
Comment (STRRL) [inline docs/release-2.3-tracking.md:18]:
```suggestion
## Summary
```

how about following https://github.com/kubernetes/sig-release/blob/master/releases/release-1.24/README.md

using "Summary" as the plain and using "Timeline" to track what actually happens

**Review by STRRL — COMMENTED** [review]
rest LGTM :)

**Review by Hexilee — APPROVED** [review]
LGTM!

**Review by STRRL — APPROVED** [review]
LGTM!

**Pull request merged**